src/table/cells/withStyling.jsx

```diff
@@ -1,35 +1,10 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { makeStyles } from '@material-ui/core/styles';
-import { STYLING_DEFAULTS } from '../styling-utils';
-
-const useStyles = makeStyles({
-  cellBackground: (props) => ({
-    background: props.background,
-  }),
-  cellStyling: (props) => ({
-    color: props.fontColor,
-    backgroundColor: props.backgroundColor,
-    fontSize: props.fontSize,
-    padding: props.padding,
-    height: STYLING_DEFAULTS.HEIGHT,
-    lineHeight: STYLING_DEFAULTS.BODY_LINE_HEIGHT,
-    '&&:focus': {
-      boxShadow: STYLING_DEFAULTS.FOCUS_OUTLINE,
-    },
-  }),
-});
 
 export default function withStyling(CellComponent) {
   const HOC = (props) => {
     const { styling } = props;
```

Comment:
Okay, now I see.  I thought `styling` here was from <TableBody><TableRow><CellRenderer styling={styling}></CellRenderer></TableRow></TableBody>. Now I realize that it can also from <CellComponent styling={selectionStyling} > and <CellComponent {...props} styling={columnStyling} />.